schema.graphql

```diff
@@ -121,3 +121,17 @@ type NativeTransfer @entity {
   transaction: Transaction!
   block: Block!
 }
+
+type Account @entity {
+  # id is the address
+  id: ID!
+  nativeBalances: [NativeBalance]! @derivedFrom(field: "account")
+}
+
+type NativeBalance @entity {
+  id: ID!
+  balanceOffset: BigInt!
+  denom: String!
+  account: Account!
+  event: Event!
```

Comment:
Can we also add fields for the tx and block like have in other entities. My thinking is that it should simplify queries by providing a shortest path to each related lower-level entity.